_j/tf_42679.md

```diff
@@ -0,0 +1,58 @@
+---
+title: TensorFlow in version 2.3 can produce misleading output summary
+---
+
+TensorFlow in version 2.3 can produce misleading output summary when
+initialized in a specific way.
+
```

Comment:
do we have an an example of that misleading output?

Reply:
In the linked issue.

Reply:
yes yes, do we want to show it on out page too? #SEO

Reply:
Added in 7cb024b